site/topics/particle-swarm-optimization/simple-pso.rst

```diff
@@ -27,6 +27,30 @@ Evaluation
 Update Velocity & Speed
 =======================
 
+* Once the random vectors for the cognitive and social terms are determined
+* Each particle's velocity is updated
+
+.. math::
+
+    \vec{v_{i}}(t+1) = \omega\vec{v_{i}}(t)
+        + c_{1}\vec{r_{1}}(\vec{p_{i}}_{best} - \vec{x_{i}}(t))
+        + c_{2}\vec{r_{2}}(\vec{g}_{best} - \vec{x_{i}}(t))
+
+
+* And with an updated velocity, the particle's position can then be updated
+
+.. math::
+
+    \vec{x_{i}}(t+1) = \vec{x_{i}}(t) + \vec{v_{i}}(t+1)
```

Comment:
Is it worth linking back to the previous lecture where this is described?

Reply:
Not a bad idea, but nah, too lazy.